docker-compose.yaml

```diff
@@ -38,4 +44,63 @@ services:
           condition: service_healthy
     ports:
       - 8080:8080
-
+  basket:
+    profiles: [integration-test]
+    image: mozmeao/basket
```

Comment:
Does this mean we're always pulling and testing with the latest image of basket?